pkgs/os-specific/linux/waydroid/default.nix

```
@@ -56,8 +58,9 @@ python3Packages.buildPythonApplication rec {
   preFixup = ''
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
 
+    patchShebangs $out/lib/waydroid/data/scripts
```

```suggestion
    patchShebangs --host $out/lib/waydroid/data/scripts
```
otherwise cross would be broken

---

Applied.